Zc-specification/sail_zc_doc.json

```diff
@@ -0,0 +1,1743 @@
+{
+  "version": 1,
+  "git": {
+    "commit": "56a15cf76a416b9ad51de947607ca2c8879d68d3",
```

Comment:
I am assuming this is the HASH of the git commit of the sail source that this JSON file was generated from, Is it on a public repo ?

Reply:
It's the git hash of the sail-riscv repository (with Nambi's code integrated). See [the documentation](https://github.com/Alasdair/asciidoctor-sail/blob/master/doc/built/sail_to_asciidoc.pdf) for details.

The general setup is agnostic as to how you want to handle keeping the documentation code in sync with the model. You can invoke Sail to generate up-to-date documentation every build, or commit a bundle with the code embedded and various hashes that let you check if it's in sync manually.